woocommerce/rest-api/Controllers/Settings.php

```diff
@@ -51,10 +51,9 @@ class Settings extends \WP_REST_Controller {
 	 */
 	public function __construct( Abstract_Settings $settings ) {
 
-		$this->settings = $settings;
+		$this->settings  = $settings;
 		$this->namespace = 'wc/v3';
-
-		// TODO: set $this->rest_base when Abstract_Settings has a get_id() method
+		$this->rest_base = "{$settings->get_id()}/settings";
```

Comment:
Is this the endpoint we want? I added a question about that to the [story](https://app.clubhouse.io/skyverge/story/32703/add-rest-handler-class#activity-33669) were we defined the initial version of the class.

Reply:
LGTM, but let's wait and see if @ChaseWiseman has any comments on it...